packages/mds-policy-service/tests/policy-repository.spec.ts

```diff
@@ -340,25 +340,30 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
     })
 
     it('can sort Policies by status', async () => {
-      const n = now()
```

Comment:
worst name for 'now' is a single letter. I don't think they're meaningful outside of a small/tight map/loop function.

Reply:
We're at "now" now.